pkg/client/kubernetes/client.go

```diff
@@ -94,7 +93,7 @@ func NewClientFromBytes(kubeconfig []byte, fns ...ConfigFunc) (Interface, error)
 // be used.
 func NewClientFromSecret(ctx context.Context, c client.Client, namespace, secretName string, fns ...ConfigFunc) (Interface, error) {
 	secret := &corev1.Secret{}
-	if err := c.Get(ctx, kutil.Key(namespace, secretName), secret); err != nil {
+	if err := c.Get(ctx, client.ObjectKey{Namespace: namespace, Name: secretName}, secret); err != nil {
```

Comment:
Why was it changed?

Reply:
It was necessary to resolve cyclic dependencies due to the introduction of `DefaultGetOptions`, `DefaultUpdateOptions`, etc. (https://github.com/gardener/gardener/blob/2fb3b07c5c741b432889ce155a89a82cd88532b1/pkg/client/kubernetes/types.go#L70-L76)